test/regress_dns.c

```diff
@@ -1183,17 +1186,27 @@ dns_initialize_nameservers_test(void *arg)
 		evdns_base_free(dns, 0);
 }
 #ifndef _WIN32
-#define RESOLV_FILE "empty-resolv.conf"
+#define RESOLV_FILE "./empty-resolv.conf"
 static void
 dns_nameservers_no_default_test(void *arg)
 {
 	struct basic_test_data *data = arg;
 	struct event_base *base = data->base;
 	struct evdns_base *dns = NULL;
 	int ok = access(RESOLV_FILE, R_OK);
+	int fd = -1;
 
 	tt_assert(ok);
 
+	/* create the empty file if the file didn't exist */
```

Comment:
Let's create this file only for the `EVDNS_ERROR_NO_NAMESERVERS_CONFIGURED` test, and I think we can separate this test case into a separate test.